builtin/types/chainconfig/chainconfig.proto

```diff
@@ -89,6 +89,41 @@ message RemoveFeatureRequest {
     repeated string names = 1;
 }
 
+message Action {
+    string name = 1; 
+    string value = 2;
+    // Build number that support this config
+    uint64 build_number = 3;
+}
+
+message AppStore {
```

Comment:
Message should be named `AppStoreConfig`, the field in the `Config` message should be named `app_store`.

Reply:
Done